roles/ceph-infra/tasks/tune_kernel_parameters.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "tune kernel parameters {{ tune_kernel_parameters }} and merge with existing templates from /etc/grub.d"
+  command: >
+    grubby
+    --update-kernel=ALL
+    --copy-default
```

Comment:
trying this option on CentOS 7 but it fails

```console
# grubby --update-kernel=ALL --copy-default --args="foo=bar"
grubby: kernel path expected
```

Without the --copy-default it's working fine.
Is that specific to RHEL 8 ?